docs/Settings.md

```diff
@@ -5932,6 +5932,16 @@ Enable workaround for early AKK SAudio-enabled VTX bug.
 
 ---
 
+### vtx_smartaudio_sixtynine
```

Comment:
@erstec let's make the name more generic like `vtx_smartaudio_stopbits` with default 2. What do you think?

Reply:
@DzikuVx sure! Updated commit forcepushed, together with rebase on actual 'master'